src/db/index.js

```diff
@@ -9,4 +9,12 @@ const pool = new Pool({
   port: config.postgres_port,
 });
 
-module.exports = pool;
+const userPool = new Pool({
```

Comment:
You need use the values from BACKSTAGE_DB_NAME, BACKSTAGE_DB_USER, BACKSTAGE_DB_PWD, BACKSTAGE_DB_HOST here